go/pkg/services/logcollector/server.go

```diff
@@ -475,6 +480,65 @@ func (s *Server) StopLog(ctx context.Context, request *protologcollector.StopLog
 	return s.successfulBaseResponse(), nil
 }
 
+// DeleteLogs deletes the log file for a given run id or project
+func (s *Server) DeleteLogs(ctx context.Context, request *protologcollector.StopLogsRequest) (*protologcollector.BaseResponse, error) {
+	if !s.isChief {
+		s.Logger.DebugWithCtx(ctx,
+			"Server is not the chief, ignoring delete logs request",
+			"project", request.Project,
+			"numRunIDs", len(request.RunUIDs))
+		return s.successfulBaseResponse(), nil
+	}
+
+	// validate project name
+	if request.Project == "" {
+		message := "Project name must be provided"
+		s.Logger.ErrorWithCtx(ctx, message)
+		return &protologcollector.BaseResponse{
+			Success:      false,
+			ErrorCode:    0,
```

Comment:
I'm not entirely into the log collector code yet but shouldn't you be using an error code like 400?